specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyModel.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {},
+    "copyRequest": {
+      "targetResourceId": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{resourceName}",
+      "targetResourceRegion": "{region}",
+      "authorization": {
+        "modelId": "{modelId}"
+      }
```

Comment:
Sense this is a single field can we flatten this and remove it completely?
For example: 

```json
{
    "targetResourceId": "\/subscriptions\/b7777967-35fa-44ed-b4d0-125820f270f1\/resourceGroups\/CogUITest\/providers\/Microsoft.CognitiveServices\/accounts\/FormUnderstandingUITest01",
    "targetResourceRegion": "westus2",
    "modelId": "14e99850-0ef6-4ecb-9391-6f124c078862"    
}
```

Reply:
Hey Assaf,
 The "modelId" in this case is an explicit claim token. i.e., the backend validates that the expected artifact of this model exists on the target before the copy is done. We might introduce other such "claim" options in future.